src/Explorer/Menus/CommandBar/CommandBarComponentAdapter.tsx

```diff
@@ -54,7 +55,9 @@ export const CommandBar: React.FC<Props> = ({ container }: Props) => {
   const uiFabricControlButtons = CommandBarUtil.convertButton(controlButtons, backgroundColor);
   uiFabricControlButtons.forEach((btn: ICommandBarItemProps) => (btn.iconOnly = true));
 
-  uiFabricControlButtons.unshift(CommandBarUtil.createConnectionStatus("connectionStatus"));
+  if (userContext.features.notebooksTemporarilyDown === false && userContext.features.phoenix === true) {
```

Comment:
>notebooksTemporarilyDown [](http://example.com/codeflow?start=27&length=24)

do we need this check?

why in some places only phoenix flag is checked and in some other places, both are checked.

Reply:
At parent level if a check is already made and there is no other reverences then i used of phoenix flag.